notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -0,0 +1,785 @@
+       "</div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ws = Workspace.from_config('/dbfs/aml_config/config.json')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the Serialized Model\n",
+    "Spark Serving needs the schema of the raw input data so an additional file is added to the model directory.<br>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<style scoped>\n",
+       "  .ansiout {\n",
+       "    display: block;\n",
+       "    unicode-bidi: embed;\n",
+       "    white-space: pre-wrap;\n",
+       "    word-wrap: break-word;\n",
+       "    word-break: break-all;\n",
+       "    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n",
+       "    font-size: 13px;\n",
+       "    color: #555;\n",
+       "    margin-left: 4px;\n",
+       "    line-height: 19px;\n",
+       "  }\n",
+       "</style>\n",
+       "<div class=\"ansiout\"></div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "### MAKE THIS METHOD PUBLIC WITH DEFAULT HEADER ###\n",
+    "raw_schema = get_spark_schema()\n",
+    "with open(os.path.join('/dbfs', MODEL_NAME, 'schema.json'), 'w') as f:\n",
+    "  f.write(raw_schema.json())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Copy from dbfs to local\n",
+    "\n",
+    "While you can access files on DBFS with local file APIs, it is safer to explicitly copy saved models to and from dbfs, because the local file APIs can only access files smaller than 2 GB (see details [here](https://docs.databricks.com/user-guide/dbfs-databricks-file-system.html#access-dbfs-using-local-file-apis))."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<style scoped>\n",
+       "  .ansiout {\n",
+       "    display: block;\n",
+       "    unicode-bidi: embed;\n",
+       "    white-space: pre-wrap;\n",
+       "    word-wrap: break-word;\n",
+       "    word-break: break-all;\n",
+       "    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n",
+       "    font-size: 13px;\n",
+       "    color: #555;\n",
+       "    margin-left: 4px;\n",
+       "    line-height: 19px;\n",
+       "  }\n",
+       "</style>\n",
+       "<div class=\"ansiout\"><span class=\"ansired\">Out[</span><span class=\"ansired\">6</span><span class=\"ansired\">]: </span>True\n",
+       "</div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "model_local = os.path.join(os.getcwd(), MODEL_NAME)\n",
+    "dbutils.fs.cp('dbfs:/' + MODEL_NAME, 'file:' + model_local, recurse=True)"
```

Comment:
Will this (using of `dbutils`) work out of box in the notebook?

Reply:
only on databricks, but that's one of the assumptions listed at the top

Reply:
I see 👍